frontend/packages/kubevirt-plugin/src/components/modals/create-vm-wizard.tsx

```diff
@@ -80,7 +81,21 @@ export const openCreateVmWizard = (activeNamespace, createTemplate = false) => {
     };
   };
 
-  const launcher = createModalResourceLauncher(CreateVmWizard, resources, resourcesToProps);
+  type Props = {
```

Comment:
Can we have the type static and not create with each call of `openCreateVmWizard` ?

Reply:
and possibly better named?